metrix-mapping/src/main/java/com/powsybl/metrix/mapping/log/AbstractLogBuilder.java

```diff
@@ -0,0 +1,8 @@
+package com.powsybl.metrix.mapping.log;
+
+public abstract class AbstractLogBuilder {
```

Comment:
QCA  : RMQ => As discuss remove this class and create a LogContent class with the label and the message